docs/sanity-migration/10-neon-integration-yep.md

```diff
@@ -0,0 +1,206 @@
+# Chapter 10: Neon Integration (Year End Poll)
+
+## Overview
+
+This chapter documents the hybrid Sanity + Neon architecture for the **Year End Poll**. This architecture splits poll data between two systems:
+
+- **Sanity CMS**: Staff-managed configuration (poll setup, categories, options)
+- **Neon PostgreSQL**: High-volume user data (votes, contest entries)
+
+This separation provides:
+- **Cost efficiency**: Thousands of votes don't impact Sanity API costs
+- **Performance**: PostgreSQL handles vote aggregation efficiently
+- **Privacy**: PII (contestant contact info) stored appropriately
+- **Scalability**: Database can handle high traffic during voting periods
+
+### Data Flow
+
+```
+┌─────────────┐         ┌──────────────┐         ┌─────────────┐
+│   Sanity    │◄────────│  Next.js API │────────►│    Neon     │
+│             │         │              │         │  PostgreSQL │
+│ • Poll      │         │ • Validate   │         │             │
+│   Config    │         │ • Authorize  │         │ • Votes     │
+│ • Categories│         │ • Process    │         │ • Entries   │
+│ • Options   │         │              │         │ • Users     │
+└─────────────┘         └──────────────┘         └─────────────┘
+```
+
+1. User visits site → Frontend fetches poll config from Sanity
+2. User submits votes → API validates against Sanity config
+3. API stores votes in Neon PostgreSQL (one vote per category)
+4. Results aggregated from Neon, matched with Sanity option data
+
+---
+
+## Sanity Schemas
+
+### `yearEndPoll`
+Annual poll configuration.
+
+**Document ID format:** `yep-{year}` (e.g., `yep-2025`)
+
+**Key fields:**
+- `title`: Display title (e.g., "2025 Year End Poll")
+- `year`: Poll year
+- `votingOpensAt`: When voting begins
+- `votingClosesAt`: When voting ends
+- `contestPrize`: Prize description for contest entry drawing
+- `status`: `draft` | `open` | `closed` | `archived`
+
+### `yearEndPollCategory`
+Individual category within the Year End Poll.
+
+**Document ID format:** `yep-{year}-{categorySlug}` (e.g., `yep-2025-song-of-the-year`)
+
+**Key fields:**
+- `poll`: Reference to `yearEndPoll` document
+- `name`: Category name (e.g., "Song of the Year")
+- `categoryType`: `song` | `album` | `artist` | `band`
+- `displayOrder`: Order in poll display
+- `maxSelections`: How many options voters can select
+- `allowWriteIns`: Whether write-in votes are permitted
+- `songOptions` / `albumOptions` / `artistOptions`: Arrays of references based on category type
```

Comment:
Options for some categories may be simple strings. For examples, movies or TV shows